Containerfile.tests

```diff
@@ -0,0 +1,30 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+FROM fedora:34
+
+WORKDIR /tokman
```

Comment:
In the test image the working directory should be `/src`.

Reply:
I think you meant to replace tokman w/ src throughout, so that's
what my latest round of changes reflects.